ibmcloud-janitor/resources/clean.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/boskos/common"
+	"sigs.k8s.io/boskos/common/ibmcloud"
+)
+
+func CleanAll(res *common.Resource, debugflag bool) error {
+	if res.Type == "" {
+		logrus.Info("No valid account selected")
+		return nil
+	} else if res.Type == PowervsResourceType {
+		powervsData, err := ibmcloud.GetResourceData(res)
+		if err != nil {
+			return errors.Wrap(err, "Failed to get the resource data")
+		}
+		powervsClient, err := NewPowerVSClient(res, powervsData, debugflag)
+		if err != nil {
+			return errors.Wrap(err, " Failed to create new PowerVS client")
+		}
+		logrus.Info("Successfully created PowerVS client!")
+
+		for _, resource := range PowervsResources {
+			err = resource.cleanup(powervsClient)
+			if err != nil {
+				return errors.Wrap(err, "Failed to clean the powervs resources")
+			}
+		}
+	}
+	return nil
```

Comment:
I see its NoOps if user specify list of resources in the command line e.g: `--resource-type=powervs-service-instance,vpc-service,classic`, nothing happens in this function with new info/warning message.

It will be good to add some warning message if user specify something else or allow user to use only the `powervs-service-instance,vpc-service`

Reply:
Changed and added a warning.